fuzz/roundtrip.cpp

```diff
@@ -496,6 +496,36 @@ extern "C" int LLVMFuzzerTestOneInput(const uint8_t *data, size_t size) {
         }
       }
     }
+
```

Comment:
I think the roundtrip fuzzer covers too many cases already, it is a bit hard to grasp when it is too long. Can it be in a separate file? I can have a look at it later once I finish what I have in flight.

Reply:
Sure. We can break it down. This can be done as a follow-up PR. :-)